src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -444,7 +433,7 @@ public String getElementAttribute(String attributeLocator) {
             "Example: ${id}=    Get Element Attribute   css:h1  id\r\n" +
             "\r\n" +
             "Passing attribute name as part of the locator was removed in SeleniumLibrary 3.2. The explicit attribute argument should be used instead.")
-    @ArgumentNames({ "locator", "attribute" })
+    @ArgumentNames({ "locator", "attribute=None" })
```

Comment:
My approach:
@ArgumentNames({ "locator", "attribute=None" })
public String getElementAttribute(String locator, String... attribute) {
String attribute= robot.getParamsValue(attribute, 0, "None");

What do you think?

Reply:
I tried this but for some reason it wouldn't except both the version with 1 parameter and the version with 2 parameters that way. That would only work with just the String... params parameter.

Reply:
Ah wait, serves me right, trying to answer stuff like this without access to the code... See the fix below.

Reply:
Thanks, great